CalendarKitDemo/CalendarApp/CustomCalendarExampleController.swift

```diff
@@ -46,73 +45,29 @@ class CustomCalendarExampleController: DayViewController, DatePickerControllerDe
                 UIColor.green,
                 UIColor.red]
 
-  lazy var customCalendar: Calendar = {
-    let customNSCalendar = NSCalendar(identifier: NSCalendar.Identifier.gregorian)!
-    customNSCalendar.timeZone = TimeZone(abbreviation: "CEST")!
-    let calendar = customNSCalendar as Calendar
-    return calendar
+  private lazy var rangeFormatter: DateIntervalFormatter = {
+    let fmt = DateIntervalFormatter()
+    fmt.dateStyle = .none
+    fmt.timeStyle = .short
+
+    return fmt
   }()
-  
+
   override func loadView() {
-    calendar = customCalendar
+    calendar.timeZone = TimeZone(identifier: "Europe/Paris")!
```

Comment:
Could you please bring back the custom calendar implementation, as it shows how to launch CK and change something else but timezone. E.g. when a completely custom calendar is needed.